app/screens/redeem-lnurl-withdrawal-screen/redeem-bitcoin-detail-screen.tsx

```diff
@@ -135,17 +140,28 @@ const RedeemBitcoinDetailScreen = ({
   navigation,
   route,
 }: StackScreenProps<RootStackParamList, "redeemBitcoinDetail">) => {
-  const { fiatSymbol } = useDisplayCurrency()
+  const { formatMoneyAmount, displayCurrency } = useDisplayCurrency()
 
   const { callback, domain, defaultDescription, k1, minWithdrawable, maxWithdrawable } =
     route.params.receiveDestination.validDestination
-  const minWithdrawableSatoshis = minWithdrawable / 1000
-  const maxWithdrawableSatoshis = maxWithdrawable / 1000
+  const minWithdrawableSatoshis: BtcPaymentAmount = {
+    amount: Math.round(minWithdrawable / 1000),
+    currency: WalletCurrency.Btc,
+  }
+  const maxWithdrawableSatoshis: BtcPaymentAmount = {
+    amount: Math.round(maxWithdrawable / 1000),
+    currency: WalletCurrency.Btc,
+  }
+  const canSetAmount = minWithdrawableSatoshis.amount !== maxWithdrawableSatoshis.amount
 
   const [receiveCurrency, setReceiveCurrency] = useState<WalletCurrency>(
     WalletCurrency.Btc,
   )
   const { LL } = useI18nContext()
+  const { data } = useReceiveBtcQuery({ fetchPolicy: "cache-first" })
+  const btcWalletId = data?.me?.defaultAccount?.btcWallet?.id
+
+  const usdWalletId = null // TODO: when usd wallet ln invoices can be generated providing the sats amount as in put we can have the usdWalletId from useMainQuery as follows: const { usdWalletId } = useMainQuery()
```

Comment:
we should not use the mainQuery on this screen. I know the comment has just been copied but it's no longer relevant